torch_audiomentations/utils/file.py

```diff
@@ -16,6 +17,11 @@ def find_audio_files(path):
     return files
 
 
-def load_audio(audio_file_path, sample_rate):
+def load_audio(audio_file_path, sample_rate=None, start=0, stop=None):
     """Loads the audio given the path of an audio file."""
-    return librosa.load(audio_file_path, sr=sample_rate)[0]
+    audio, source_sample_rate = soundfile.read(audio_file_path, start=start, stop=stop)
```

Comment:
Another question I don't have the answer to. 
Should start and stop be wrt the specified sample rate or the source sample rate? 
It seems more natural to consider the specified sample rate IMO.

Reply:
Good point. As it stands, it's also not entirely clear if start and stop is in samples or in seconds